pegasus/sites.v3/code.org/public/css/hoc-banner.scss

```diff
@@ -1,48 +1,110 @@
 @import 'color.scss', 'font.scss', 'imports/_breakpoints.scss';
 
 // Code.org HoC homepage banner
-.hoc-banner.homepage {
-  text-align: center;
-  color: $darkest_gray;
-  padding: 7em 0;
-  line-height: 1.4;
+#bars {
```

Comment:
This code is referencing this: https://github.com/code-dot-org/code-dot-org/blob/c03e5c64dc0631d29380b536f70e705baa05da9d/pegasus/sites.v3/code.org/views/homepage_hero.haml#L95-L99